hist/histpainter/src/TPaletteAxis.cxx

```diff
@@ -406,7 +406,8 @@ void TPaletteAxis::Paint(Option_t *)
    Double_t wmax = fH->GetMaximum();
    Double_t wlmin = wmin;
    Double_t wlmax = wmax;
-   Double_t y1, y2, w1, w2, zc;
+   Double_t b1, b2, w1, w2, zc;
+   Double_t kHorizontal = false;
```

Comment:
Should be "Bool_t" instead of "Double_t", I think.

Reply:
Oups .... thanks